objectbox/test/boxnn_test.dart

```diff
@@ -0,0 +1,88 @@
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:test/test.dart';
+import 'package:objectbox/objectbox.dart';
+import 'entity.dart';
+import 'entity2.dart';
+import 'test_env.dart';
+
+// Tests non-nullable fields
+void main() {
+  late TestEnv env;
+  late Box<TestEntityNN> box;
+
+  setUp(() {
+    env = TestEnv('box-nn');
+    box = env.store.box();
+  });
+
+  tearDown(() => env.close());
+
+  test('CRUD', () {
+    final objects = [
+      TestEntityNN(),
```

Comment:
Maybe change the `NN` (and `nn`) to at least `NonNull` or similar? Was kind of confused by this as I thought of N as a number placeholder.

Reply:
Ok